sc/src/test/java/uk/ac/ebi/atlas/download/ExperimentFileLocationServiceIT.java

```diff
@@ -192,7 +236,8 @@ private void existingFileOfType(String experimentAccession, ExperimentFileType f
 
     private void existingArchiveFilesOfType(String experimentAccession,
                                             ExperimentFileType fileType,
-                                            List<String> expectedFileNames) {
+                                            List<String> expectedFileNames,
+                                            Boolean folder) {
```

Comment:
What does `folder` mean? Make sure you name your variables clearly. Also, we generally name booleans like `isSomething` or `hasSomething`.